pep-0501.txt

```diff
@@ -501,11 +653,12 @@ a creating a new kind of object for later consumption by interpolation
 functions. Creating a rich descriptive object with a useful default renderer
 made it much easier to support customisation of the semantics of interpolation.
 
-Building atop PEP 498, rather than competing with it
-----------------------------------------------------
+Building atop PEP 701 rather than competing with PEP 498
+--------------------------------------------------------
+
 Earlier versions of this PEP attempted to serve as a complete substitute for
-:pep:`498`, rather than building a more flexible delayed rendering capability on
-top of :pep:`498`'s eager rendering.
+:pep:`498` . With the introduction of 701, this pep can now build a more flexible delayed
+rendering capability on top of :pep:`498` and :pep:`701`'s eager rendering.
```

Comment:
```suggestion
:pep:`498` (f-strings) . With the acceptance of that PEP and the more recent :pep:`701`,
this PEP can now build a more flexible delayed rendering capability
on top of the existing f-string eager rendering.
```

* Fix multiple grammar and phrasing issues
* Avoid repeating the PEPs over and over and instead refer to the feature names